src/products/mozilla_org/testing.md

```diff
@@ -60,7 +60,7 @@ Updated translations are pushed to the production server regularly throughout th
 
 * New page: When a brand new page is available for localization, it will be enabled on production when it’s 80% complete. At that point, if it doesn’t reach 100% completion level, the page will be available on production with English content mixed with translation. Before the page is activated in production, use the staging server for testing.
 * Activated page with new string update (no tags): The strings will appear in English on production right away. Localized content will be pushed to production as soon as translations are available in the repository.
-* Activated page with tagged new string update: The switch to the updated content happens when tagged strings are fully localized. In some cases, there is a defined period of time frame to allow tagged strings to be fully localized. Test the updates on staging server.
+* Activated page with tagged new string update: The switch to the updated content happens when tagged strings are fully localized. In some cases, there is a defined period of time frame to allow tagged strings to be fully localized. Test the updates on staging server. The obsolete string will be removed three months after the introduction of the new string update.
```

Comment:
(comment valid for this and the previous comment with "no tags") 

Again, there is no concept of tagging a string in current Bedrock. That refers to a very specific action when we used langchecker. 

There are new strings, and there are obsolete strings (which have a comment calling out them as such).
New strings are used when they're translated, fallbacks (obsolete) are used in the meantime, until they're removed (3 months).

Similarly, there are no "activated pages" anymore, that refers to the `## active ##`  tags.

Reply:
I reworded this section and distinguish the types of new content: 1) new page, 2). page with new strings, 3). page with new strings to replace obsolete strings.